rosdep/python.yaml

```diff
@@ -6481,6 +6481,9 @@ python3-construct:
   fedora: [python3-construct]
   gentoo: [dev-python/construct]
   nixos: [python3Packages.construct]
+  rhel:
+    '*': [python3-construct]
+    '7': null
```

Comment:
A non-blocking question, but do we even care about RHEL-7 keys at this point?

Reply:
I figured we'd drop it when we add RHEL 9.